tools/addon_updater.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python3
```

Comment:
Should we add this to the readme or the make file? I mean it's just calling this script with python, but it might be nice to document it's existence

Reply:
will do